python/Ganga/Core/GangaRepository/GangaRepositorySQLite.py

```diff
@@ -34,7 +34,7 @@ def startup(self):
             os.makedirs(self.root)
         except OSError as x:
             pass
-        self.con = sqlite.connect(os.path.join(self.root, "database.db"))
+        self.con = sqlite3.connect(os.path.join(self.root, "database.db"))
```

Comment:
Well `sqlite` does not exist and so when pytest is searching the codebase to look for tests it tries to import this module and fails. This change was just to get it importable.